flink-runtime-web/web-dashboard/package.json

```diff
@@ -43,11 +45,11 @@
     "husky": "^1.3.1",
     "jasmine-core": "~2.99.1",
     "jasmine-spec-reporter": "~4.2.1",
+    "karma": "~4.0.0",
```

Comment:
Reordering probably makes it much harder to track what packages you changed here.

Reply:
Thanks for spotting. Seems that my editor applied different ordering rules for strings of variable length than what was in the file originally.